CONTRIBUTING.md

```diff
@@ -66,6 +67,112 @@ please ensure that:
+
+The code-review process is a normal part of software development, and nothing to
+be scared of; for very easy changes it can be as simple as a maintainer saying
+"looks good to me!" (or in short-form, "LGTM!") and merging the PR.  For more complex changes, it's often a
+back-and-forth where the reviewer may ask a couple of questions about why things
+were done a particular way, and make suggestions for improvement.  You don't
+need to do everything suggested if you've got good reasons to disagree, but
+communicate that clearly and politely.
+
+If you're struggling with code review or a PR on Qiskit, you can ask for help in
+the `#qiskit-pr-help` channel on [the public Qiskit Slack](https://qisk.it/join-slack).
+
+Remember that the PR author is a human, not just a username!  It's ok to ask
+questions about the code, but don't be mean or rude about it even if you don't
+like it.
+
+#### What to focus on in review
+
+* Is everything in [the PR checklist](#pull-request-checklist) done?
+* Are any new public APIs easy to use, well documented, and consistent with
+  other parts of the Qiskit API?
+* Do any changes to the code have knock-on effects for other parts of Qiskit
+  that may be using them, or do they imply changes to the assumptions in
+  our data structures?
+* Are there any edge cases you can think of that the code might not handle well?
+* Is the code reasonably easy for you to understand?  This particular point is
+  tricky; the more you review code, the easier it will be for you to understand
+  more code, so don't worry about this as much if you're getting started.
+
+#### Writing review comments
+
+* Make concrete suggestions when you think something should be changed, but
+  remember that the author might have already thought about it and have a
+  reason.  Try "what do you think about us raising a `TypeError` here instead of
+  returning `None`?", rather than "you should raise an exception here".
+* Try to make each round of review thorough.  Don't add one or two comments on
+  one file, then come back a day later and add a couple of other unrelated
+  comments on a different file, and so on.  Try to review the whole PR
+  thoroughly in one go; it's easier to catch bugs like this, and less
+  frustrating to be on the other side of. If that's too much for you, consider
+  asking if the PR could be split into smaller independent chunks.
+* Try to keep the number of comments reasonable.  This depends on the size of
+  the PR, but remember that there's somebody who'll read all your comments, and
+  it can be demoralizing if you get a PR back and it's got 30 comments on from a
+  50-line change.  If you feel like you're putting too many comments on,
+  consider if you could group several of them into one theme, and ask them as a
+  more detailed question with a focus on only one part of the code.  Try not to
+  comment the same thing in many places.
+* Try to avoid saying "you do" in review comments, and instead try to
```

Comment:
I'm a bit confused about this you do / we do comment, I'm not sure what exactly you're trying to say 😅 Maybe add an example?